### ISSUE TITLE
Add link to Integration documentation

### ISSUE THREAD

**Issue by andriej** [home-assistant/frontend#5572]
## The request

It would be very good to have Integration's documentation link while adding/configuring it, just as like it is with HA addons - there's always link at top and it makes finding proper docs very easy.
## The alternatives
n/a

## Additional information

It would be good to have it at least while configuring new Integration.

**Comment by SeanPM5** [comment]
That reminds me, I did a rough mockup for this a year ago (also logos).

![image](https://camo.githubusercontent.com/9e3abc675e1dfb1d8d3fb116be35cfc8cd67b8b0/68747470733a2f2f692e696d6775722e636f6d2f705930646252542e6a7067)

I agree this would be a good idea since there can often be some useful information / links in the docs about how to register for developer accounts and obtain API keys and whatnot. 

However the one thing that gives me a little pause though, is that at present time the docs are _very_ YAML oriented. If someone is adding it through the UI, then clicks a docs link for help and there's all this stuff about YAML, they could be confused and second guessing themselves? That will be less of a problem over time, just one potential drawback to consider right now.

**Comment by andriej** [comment]
Yes, I agree - it would need different approach to docs also... but maybe it's worth a try?
